clang/test/lab1/safronov_mikhail/test.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: not %clang_cc1 -load %llvmshlibdir/PrintClassesPlugin%pluginext -plugin print-classes --help %s 2>&1 | FileCheck %s
```

Comment:
It would be nice to split test for main fuctionality and help argument check.

Reply:
it is not working:'(